slave-dotnet10/Dockerfile.rhel7

```diff
@@ -0,0 +1,44 @@
+FROM openshift/jenkins-slave-base-rhel7
+
+# Labels consumed by Red Hat build service
+LABEL com.redhat.component="jenkins-slave-dotnet10-rhel7-docker" \
+      name="openshift3/jenkins-slave-dotnet10-rhel7" \
+      version="3.6" \
+      architecture="x86_64" \
+      release="1" \
+      io.k8s.display-name="Jenkins Slave .NET Core 1.0" \
+      io.k8s.description="The jenkins slave dotnet image has the dotnet tools on top of the jenkins slave base image." \
+      io.openshift.tags="openshift,jenkins,slave,dotnet,dotnet10"
+
+ENV BASH_ENV=/usr/local/bin/scl_enable \
+    ENV=/usr/local/bin/scl_enable \
+    PROMPT_COMMAND=". /usr/local/bin/scl_enable"
+
+COPY contrib/bin/scl_enable /usr/local/bin/scl_enable
+
+# Install .NET Core
+RUN yum-config-manager --enable rhel-7-server-dotnet-rpms && \
+    yum-config-manager --enable rhel-server-rhscl-7-rpms && \
+    yum-config-manager --disable epel >/dev/null || : && \
+    INSTALL_PKGS="rh-dotnetcore10 rh-nodejs4-npm" && \
+    yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
+    rpm -V $INSTALL_PKGS && \
+    yum clean all -y
+
+# Don't download/extract docs for nuget packages
+# Don't do initially populate of package cache
+ENV NUGET_XMLDOC_MODE=skip \
+    DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
```

Comment:
can all these ENVs be combined into a single ENV line?  it'll save a layer/build time.